scripts/force-lowest-dependencies.js

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
Would it be worth to put this into a separate npm package? This sounds like something that more developers would run into when maintaining packages.

Reply:
as this is meant to run before a `npm install`, it might be harder :smile:

Reply:
Maybe something like a global install? Or a phar like equivalent for js

Reply:
I'm always interested in re-using existing libs or extracting Encore into new libs. But, with limited resources, we need to stay focused on Encore itself (i.e. things like this would need to be done by some community member).